flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/functions/utils/UserDefinedFunctionUtils.scala

```diff
@@ -733,6 +733,10 @@ object UserDefinedFunctionUtils {
         candidate == classOf[Timestamp] && expected == classOf[SqlTimestamp] ||
         candidate == classOf[SqlTimestamp] && expected == classOf[Timestamp] ||
         candidate == classOf[LocalDateTime] && expected == classOf[SqlTimestamp] ||
+        candidate == classOf[SqlTimestamp] && expected == classOf[Instant] ||
+        candidate == classOf[Instant] && expected == classOf[SqlTimestamp] ||
+        candidate == classOf[Instant] && (expected == classOf[Long] ||
```

Comment:
Do we need add here? Can we remove these?

Reply:
removed